docs/resources/cbr_vault.md

```diff
@@ -103,42 +115,49 @@ The following arguments are supported:
+  + **[app_consistent](https://support.huaweicloud.com/intl/en-us/usermanual-cbr/cbr_03_0109.html)**
 
-* `consistent_level` - (Required, String, ForceNew) Specifies the backup specifications. The valid values
-  are *[crash_consistent](https://support.huaweicloud.com/intl/en-us/usermanual-cbr/cbr_03_0109.html)*
-  and *[app_consistent](https://support.huaweicloud.com/intl/en-us/usermanual-cbr/cbr_03_0109.html)*
   Only server type vaults support application consistent. Changing this will create a new vault.
 
-* `protection_type` - (Required, String, ForceNew) Specifies the protection type of the CBR vault. The valid values are
-  backup and replication. Vaults of type disk don't support replication. Changing this will create a new vault.
+* `protection_type` - (Required, String, ForceNew) Specifies the protection type of the CBR vault.
+  The valid values are **backup** and **replication**. Vaults of type **disk** don't support **replication**.
+  Changing this will create a new vault.
 
-* `size` - (Required, Int) Specifies the vault sapacity, in GB. The valid value is ranges from 1 to 10485760.
+* `size` - (Required, Int) Specifies the vault sapacity, in GB. The valid value range is `1` to `10,485,760`.
 
 * `auto_expand` - (Optional, Bool) Specifies to enable auto capacity expansion for the backup protection type vault.
-  Default to false.
+  Default to **false**.
 
 * `enterprise_project_id` - (Optional, String, ForceNew) Specifies a unique ID in UUID format of enterprise project.
   Changing this will create a new vault.
 
 * `policy_id` - (Optional, String) Specifies a policy to associate with the CBR vault.
   `policy_id` cannot be used with the vault of replicate protection type.
 
-* `resources` - (Optional, List) Specifies an array of one or more resources to attach to the CBR vault. The resources
-  structure is documented below.
+* `resources` - (Optional, List) Specifies an array of one or more resources to attach to the CBR vault.
+  The [object](#cbr_vault_resources) structure is documented below.
 
 * `tags` - (Optional, Map) Specifies the key/value pairs to associate with the CBR vault.
 
+<a name="cbr_vault_resources"></a>
 The `resources` block supports:
 
-* `id` - (Required, String) Specifies the ID of the resource to be backed up.
+* `id` - (Optional, String) Specifies the ID of the resource to be backed up.
 
-* `exclude_volumes` - (Optional, List) Specifies the disks included in the backup except for the array that contains one
-  or more disk IDs.
+* `excludes` - (Optional, List) Specifies the disks included in the backup except for the array that contains one or
+  more disk IDs. Only **server** vault support this parameter.
 
-* `include_volumes` - (Optional, List) Specifies the array of disk IDs which will be included in the backup. This
-  parameter and `exclude_volumes` are alternative.
+* `includes` - (Optional, List) Specifies the array of disk or SFS file system IDs which will be included in the backup.
```

Comment:
why `server` vault can't support this?

Reply:
For API limit, the `includes` (whitelist) only provided for `disk` and `sfs turbo`.

Reply:
ok